docs/javascript/browser-apis-and-jquery.md

```diff
@@ -224,6 +224,125 @@ The `document.getElementById()` method allows you to select an HTML element by i
 <figcaption></figcaption>
 </figure>
 
+### Getting an element by `class name`
+
+The `document.getElementsByClassName()` method allows you to return the selected HTML elements by it's class attribute:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div>
+      <div class="square"></div>
+      <div class="circle"></div>
+      <div class="square"></div>
+      <div class="square"></div>
+      <div class="circle"></div>
+    </div>
+    <script>
+      // Return all elements with the class name of `square`
+      var getSquares = document.getElementsByClassName("square");
+
+      // Add an image to all div's with class name `square`
+      getSquares.style.backgroundImage = "<img src='square.png'>";
```

Comment:
This won't actually work, because `getSquares` is an HTMLCollection (kind of like an array). 

Here's an alternative:

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Home</title>
    <style>
        .square {
            width: 300px;
            height: 300px;
            margin: 24px;
            background-repeat: no-repeat;
        }
    </style>
  </head>

  <body>
    <div>
      <div class="square"></div>
      <div class="circle"></div>
      <div class="square"></div>
      <div class="square"></div>
      <div class="circle"></div>
    </div>
    
    <script>

      // Return all elements with the class name of `square`
      var squares = document.getElementsByClassName("square");
    
      // Add an image to all div's with class name `square`
      for (var square of squares) {
          square.style.backgroundImage = "url(square.png)";
      }

    </script>
  
</body>
</html>
```